src/store/actions/authActions.js

```diff
@@ -29,6 +32,9 @@ export const signOut = () => {
       .auth()
       .signOut()
       .then(() => {
+        // Go offline --> Writes "offline" into user status
```

Comment:
This is not the norm.
```suggestion
        // Go offline. This writes "offline" into user status.
```